velox/serializers/PrestoSerializer.cpp

```diff
@@ -1595,9 +1595,12 @@ class PrestoVectorSerializer : public VectorSerializer {
       std::shared_ptr<const RowType> rowType,
       int32_t numRows,
       StreamArena* streamArena,
-      bool useLosslessTimestamp) {
+      bool useLosslessTimestamp,
+      dwio::common::CompressionKind compressionKind) {
```

Comment:
Can make streamArena_ and codec_ const members of PrestoVectorSerializer? Thanks!

Reply:
If make codec_ const, I will receive error:
```
no operator "=" matches these operandsC/C++(349)
PrestoSerializer.cpp(1617, 12): operand types are: const std::unique_ptr<folly::io::Codec, std::default_delete<folly::io::Codec>> = std::unique_ptr<folly::io::Codec, std::default_delete<folly::io::Codec>>
```

Reply:
You put this in ctor initializer list?
```
codec_(xxx)
```